generated/nifgen/nifgen/_grpc_stub_interpreter.py

```diff
@@ -8,16 +8,18 @@
 
 from . import enums as enums
 from . import errors as errors
+from . import grpc_session_options as grpc_session_options
 from . import nifgen_pb2 as grpc_types
 from . import nifgen_pb2_grpc as nifgen_grpc
 
 
 class GrpcStubInterpreter(object):
     '''Interpreter for interacting with a gRPC Stub class'''
 
-    def __init__(self, grpc_channel):
+    def __init__(self, grpc_options):
+        self._grpc_options = grpc_options
         self._lock = threading.RLock()
-        self._client = nifgen_grpc.NiFgenStub(grpc_channel)
+        self._client = nifgen_grpc.NiFgenStub(grpc_options.grpc_channel)
```

Comment:
Note that the initialization method on line ~335 _does not use_ the session name and initialize behavior yet because I only modified the metadata for nifake. Once I have the direction approved, I will use actual exports for both.

Reply:
See [my ni-central PR](https://ni.visualstudio.com/DevCentral/_git/ni-central/pullrequest/382959) where once submitted, that will allow me to update the metadata here with an official export.

Reply:
Resolved with the latest nifgen exports.